src/amo/components/VPNPromoBanner/index.js

```diff
@@ -52,9 +55,27 @@ type InternalProps = {|
   location: ReactRouterLocationType,
 |};
 
+export const getImpressionCount = (_window: typeof window): number => {
+  const impressionCount = _window.localStorage.getItem(IMPRESSION_COUNT_KEY);
+  const parsedCount = parseInt(impressionCount || 0, 10);
+
+  if (Number.isNaN(parsedCount)) {
+    throw new Error(
+      `A non-number was stored in ${IMPRESSION_COUNT_KEY}: ${impressionCount}`,
+    );
+  }
+
+  return parsedCount;
+};
+
+export const clearImpressionCount = (_window: typeof window) => {
+  _window.localStorage.removeItem(IMPRESSION_COUNT_KEY);
```

Comment:
would using our `localState` make sense here? https://github.com/mozilla/addons-frontend/blob/dab5ceef5418ababacccb65359c9f9b816bda7f3/src/amo/localState.js

Reply:
I looked into it and I think it's overkill. `localState` seems to be designed for keeping track of data for individual entities (e.g., storing the text of a review for different add-ons - the text for each add-on is stored separately). In this case we really only need one global value for the whole site, so we don't need any of the functionality that `localState` gives us. I actually started by setting it up for the component, and I think the code ends up being more complex by trying to use `localState` and doesn't buy us anything.